### PR TITLE
Fix CFN node not showing Lambdas

### PR DESCRIPTION
**Pull request by abrooksv**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
CFN StackResourceSummary returns the function name, not the arn

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/resources/LambdaResources.kt [hunk 1]
```diff
@@ -13,5 +13,5 @@ object LambdaResources {
         listFunctionsPaginator().functions().filterNotNull().toList()
     }
 
-    fun function(arn: String) = LIST_FUNCTIONS.find { it.functionArn() == arn }
+    fun function(name: String) = LIST_FUNCTIONS.find { it.functionName() == name }
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/resources/LambdaResources.kt]: were there are places this was used?

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/resources/LambdaResources.kt]: Screen Shot 2019-09-23 at 12 33 56 PM

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/resources/LambdaResources.kt]: yup just ran that :P

**Comment by abrooksv** [comment]
Output from CLI showing it is not an ARN

```
        {
            "LogicalResourceId": "CloseReleas******",
            "PhysicalResourceId": "sam-dev-AWSToolk-CloseReleas******",
            "ResourceType": "AWS::IAM::Role",
            "LastUpdatedTimestamp": "2019-05-28T17:08:21.232Z",
            "ResourceStatus": "CREATE_COMPLETE",
            "DriftInformation": {
                "StackResourceDriftStatus": "NOT_CHECKED"
            }
        },
```

**Pull request merged**